lib/modules/platform/pr-body.ts

```diff
@@ -9,6 +10,14 @@ export const prDebugDataRe = regEx(
   /\n?<!--renovate-debug:(?<payload>.*?)-->\n?/
 );
 
+export const renovateConfigHashRe = regEx(
+  /\n?<!--renovate-config-hash:(?<payload>.*?)-->\n?/
+);
+
+export const prCheckboxRe = regEx(
+  /- (?<checkbox>\[[\sx]]) <!-- rebase-check -->/
```

Comment:
i don't think we need the extra capture group. what's the benefit? i think a test instead of exec will be faster, as we don't need any position information.

why it's exported?

do we really use the unchecked match?

Reply:
https://github.com/renovatebot/renovate/pull/17633#discussion_r964494404